src/Libraries/Microsoft.Extensions.Caching.Hybrid/Internal/DefaultJsonSerializerFactory.cs

```diff
@@ -10,6 +10,10 @@
 using System.Text.Json;
 using Microsoft.Extensions.DependencyInjection;
 
+[assembly: UnconditionalSuppressMessage("AOT", "IL2026", Justification = "Checked at runtime, guidance issued")]
+[assembly: UnconditionalSuppressMessage("AOT", "IL2070", Justification = "Checked at runtime, guidance issued")]
+[assembly: UnconditionalSuppressMessage("AOT", "IL3050", Justification = "Checked at runtime, guidance issued")]
```

Comment:
```suggestion
```

We shouldn't be suppressing these warnings for the whole assembly. Instead, we should just add the suppressions in the exact places we know are OK to suppress.